### PR TITLE
Chart focus customisation

### PR DESCRIPTION
**Pull request by ImogenF**
https://visual-tests-focus-customisation.fe.ctmo.io

### CODE REVIEW

**Comment by ImogenF** [comment]
[Charts - customisable focus labels](https://trello.com/c/VB2HXoUe/188-charts-customisable-focus-labels)

**Comment by micha-f** [comment]
screen shot 2018-06-13 at 10 41 34

@ImogenF  There seems to be a problem with the formatting of the numbers in the label.
Also - can we add the country to the label?

**Comment by ImogenF** [comment]
@micha-f is that in the website example or in the visual specs? Either way, that should be fixed...  Have just updated the deployed visual tests. 

In terms of number formatting though, the default formatter basically assumes there won't be more than 3 decimal places. It formats everything after the decimal exactly the same as everything before. I'm not sure how much logic we want to have in the default number formatter...  should I add default rounding?

Will add the countries now.

File: packages/visualizations/src/Chart/facade.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ const defaultConfig = (): ChartConfig => {
     maxBarWidthRatio: 1 / 3,
     maxFocusLabelWidth: 350,
     minBarWidth: 3,
-    numberFormatter: (x: number): string => x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ","),
+    numberFormatter: (x: number): string => (+x.toFixed(2)).toString().replace(/\B(?=(\d{3})+(?!\d))/g, ","),
```
Comment (micha-f) [inline packages/visualizations/src/Chart/facade.ts]: Is there a reason not to use `d3.format` ?

Comment (ImogenF) [reply packages/visualizations/src/Chart/facade.ts]: No, no reason

Comment (micha-f) [reply packages/visualizations/src/Chart/facade.ts]: It might be more robust / simpler than writing our own code here.

**Pull request merged**